Gruntfile.js

```diff
@@ -133,9 +133,21 @@ module.exports = function(grunt) {
       return false;
     }
 
-    var data = grunt.file.readJSON(path);
-    var tournaments = data;
+    var tournaments = grunt.file.readJSON(path);
     var players = {};
+    var calculateFinish = function(index, team, team2hg) {
+      var persons;
+
+      if (team && team2hg) {
+        persons = 2;
+      } else if (team) {
+        persons = 3;
+      } else {
+        persons = 1;
+      }
+
+      return Math.floor(index / persons) + 1;
```

Comment:
We have some similar logic in the app display code, so we could consider making a helper.